src/components/HelpIcon.jsx

```diff
@@ -1,35 +1,34 @@
-"use strict";
-
 import React from "react";
-import { OverlayTrigger, Tooltip } from "react-bootstrap";
+import { Tooltip } from "react-tooltip";
 import PropTypes from "prop-types";
-import { FaEnvelope, FaQuestionCircle } from "react-icons/fa";
-import { FaCheck } from "react-icons/fa";
-import { FaTimes } from "react-icons/fa";
-import { FaTasks } from "react-icons/fa";
+import { FaEnvelope, FaQuestionCircle, FaCheck, FaTimes, FaTasks } from "react-icons/fa";
 
 const HelpIcon = (props) => {
-  const tooltip = <Tooltip id="help-tooltip">{props.text}</Tooltip>;
-
   const icon = () => {
     switch (props.glyph) {
       case "ok":
-        return <FaCheck className={"ok-icon " + props.className} />;
+        return <FaCheck className={`ok-icon ${props.className}`} />;
       case "remove":
-        return <FaTimes className={"remove-icon " + props.className} />;
+        return <FaTimes className={`remove-icon ${props.className}`} />;
       case "to-do":
-        return <FaTasks className={"to-do-icon " + props.className} />;
+        return <FaTasks className={`to-do-icon ${props.className}`} />;
       case "envelope":
-        return <FaEnvelope className={"publish-icon " + props.className} />;
+        return <FaEnvelope className={`publish-icon ${props.className}`} />;
       default:
-        return <FaQuestionCircle className={"help-icon " + props.className} />;
+        return <FaQuestionCircle className={`help-icon ${props.className}`} />;
     }
   };
 
   return (
-    <OverlayTrigger placement="right" overlay={tooltip}>
-      {icon()}
-    </OverlayTrigger>
+    <>
```

Comment:
The new source code looks kinda complicated... Why we had to do this? OverlayTrigger does not exist anymore?

Reply:
Or Bootstrap 5 does not support tooltips anymore?

Reply:
By the way I see:
`import { OverlayTrigger, Popover, Table } from "react-bootstrap";`

in file https://github.com/kbss-cvut/record-manager-ui/blob/64ff5e3c96d988a20e8effcd585ff9958777122f/src/components/record/RecordTable.jsx#L2

How is it related? Isn't `Popover` alternative to tooltips? If so we should definitely unify the way.

Reply:
I was struggling with OverlayTrigger. After migrating, the tooltip was rendering outside of the window, and I couldn't fix its position. In the latest commit, I replaced OverlayTrigger with the Overlay component from Bootstrap, which resolved the issue. I also removed the popover.